apps/src/lib/util/javascriptMode.js

```diff
@@ -65,11 +65,15 @@ export function apiValidateType(
         break;
       case 'pinid':
         var reservedPins = ['A2', 'A3', 'A7', 1, 9, 10];
-        properType =
-          (typeof varValue === 'string' || typeof varValue === 'number') &&
-          !reservedPins.includes(varValue);
+        var validPins = [0, 2, 3, 6, 12, 'A0', 'A1', 'A4', 'A5', 'A6'];
```

Comment:
Nit: Could we please consistently list either the Classic or Express pin ids first? Compare with the line above.

Reply:
Got it!